resolwe_bio/management/commands/generate_samples.py

```diff
@@ -264,9 +271,9 @@ def create_data(self, reads_name='seq_reads', annotated=False, rseed=None):
             sample.descriptor = self.generate_sample_desciptor(d.name)
             sample.presample = False
             sample.save()
-            print "Created sample: {} (id={})".format(sample.name, sample.id)
+            self.stdout.write("Created sample: {} (id={})".format(sample.name, sample.id))
```

Comment:
Also, maybe just writing "XXX samples successfully created" would be more appropriate then a line for each sample. This gets ugly and slow when you create tens of thousands of test sample data. Test it and decide.